diff -rupN wildmidi-wildmidi-0.4.6.old/CMakeLists.txt wildmidi-wildmidi-0.4.6/CMakeLists.txt
--- wildmidi-wildmidi-0.4.6.old/CMakeLists.txt	2024-04-11 02:11:10.000000000 +0200
+++ wildmidi-wildmidi-0.4.6/CMakeLists.txt	2025-07-17 17:31:18.978876624 +0200
@@ -1,5 +1,5 @@
 # Project setup
-CMAKE_MINIMUM_REQUIRED(VERSION 3.4...3.5)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.4...3.10)
 PROJECT(wildmidi LANGUAGES C VERSION 0.4.6)
 
 # WildMIDI Version
